### PR TITLE
Change the criticality to a float number

### PR DESCRIPTION
**Pull request by Cheshiriks**

### CODE REVIEW

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/CreateVulnerabilityView.kt [hunk 1]
```diff
@@ -301,17 +301,17 @@ val createVulnerabilityView: VFC = VFC {
                                             required = true
                                             defaultValue = 0
                                             onChange = { event ->
-                                                val progress = event.target.value.toIntOrNull()
-                                                if (progress !in (1..100)) {
-                                                    setConflictErrorMessage("Progress must be an integer number from 1 to 100")
+                                                val progress = event.target.value.toFloatOrNull()
+                                                if (progress!! !in (0f..10f)) {
```
Comment (akuleshov7) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/CreateVulnerabilityView.kt:305]: not sure if Float Range will work, hope you have tested it

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/CreateVulnerabilityView.kt [hunk 2]
```diff
@@ -301,17 +301,17 @@ val createVulnerabilityView: VFC = VFC {
...
+                                                    setConflictErrorMessage("Progress must be an float number with one decimal place from 0 to 10.0")
```
Comment (akuleshov7) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/CreateVulnerabilityView.kt:306]: decimal place? -> decimal digit?

Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/CreateVulnerabilityView.kt:306]: Only one digit after the decimal point

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/CreateVulnerabilityView.kt (hunk 1 above)
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/CreateVulnerabilityView.kt]: Yes. Tested it

**Review by nulls — APPROVED** [review]

**Pull request merged**